tools/pylib/boutdata/data.py

```diff
@@ -516,6 +516,9 @@ class BoutOutputs(object):
 
     >>> d = BoutOutputs(".", prefix="BOUT.dmp", caching=True) # Turn on caching
 
+
+    :ivar attributes: dictionary of attributes from the first DataFile
+    :ivar evolvingVariables: list of time-evolving variables
```

Comment:
Please could you put these in [numpydoc][1] style? These can go under an `Attributes` section (see link)

[1]: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring